assets/settings/default.json

```diff
@@ -223,6 +223,8 @@
     // Whether to show diagnostic indicators in the scrollbar.
     "diagnostics": true
   },
+  // Enable middle-click paste on Linux.
+  "linux_middle_click_paste": true,
```

Comment:
I don't want to bikeshed this, but: can we find a better name? 

For example: both of these seem like they're kinda in the same ballpark

https://github.com/zed-industries/zed/blob/4347bdb55221639ad78c9153a658926eab8f7192/assets/settings/default.json#L646-L656

But the first one isn't prefixed with `macos`. So should we drop the `linux_` from the name?

Reply:
I don't mind that, as long as it's documented somewhere that this is a linux specific thing.  I do think it could be confusing if people try to enable this on Mac OS or Windows and it isn't implemented.  But their confusion would be nothing compared to sorting through nested pastes of random clipboard garbage every few minutes!